### PR TITLE
repo: Dev v1.23.9

### PR DESCRIPTION
**Pull request by phlax**
Commit Message:
Additional Description:
Risk Level:
Testing:
Docs Changes:
Release Notes:
Platform Specific Features:
[Optional Runtime guard:]
[Optional Fixes #Issue]
[Optional Fixes commit #PR or SHA]
[Optional Deprecated:]
[Optional [API Considerations](https://github.com/envoyproxy/envoy/blob/main/api/review_checklist.md):]

### CODE REVIEW

File: .azure-pipelines/pipelines.yml [hunk 1]
```diff
@@ -253,7 +253,7 @@ stages:
   jobs:
   - job: release
     timeoutInMinutes: 120
-    pool: x64-large
+    pool: envoy-x64-large
```
Comment (wbpcode) [inline .azure-pipelines/pipelines.yml:256]: where we can know the pool  name is changed?

**Review by wbpcode — APPROVED** [review]
LGTM. Thanks.

File: .azure-pipelines/pipelines.yml (hunk 1 above)
Comment (phlax) [reply .azure-pipelines/pipelines.yml:256]: hmm - good q - @mum4k and i changed them at the end of last week - probably you could figure that info out from https://github.com/envoyproxy/ci-infra/ - its not something that should change too often

Comment (phlax) [reply .azure-pipelines/pipelines.yml:256]: relatedly - we are looking at separating some of the pools, so i will most likely make this dynamic soon

**Pull request merged**